### PR TITLE
Bump to v1.21.0

### PR DESCRIPTION
**Pull request by prologic**

### CODE REVIEW

File: bump_v1.21.0 [hunk 1]
```diff
@@ -0,0 +1,41 @@
+diff --git a/Chart.yaml b/Chart.yaml
+index 15c5d9e..6592b03 100644
```
Comment (ilyam8) [inline bump_v1.21.0:2]: What is this diff? What purpose does it serve? 🤔

Comment (prologic) [reply bump_v1.21.0:2]: Ooops :) Removed!

**Comment by ilyam8** [comment]
i checked [prev version update PR](https://github.com/netdata/helmchart/pull/73/files), this one does same so lgtm 👍

**Comment by ilyam8** [comment]
btw @prologic i suggest to make `squash and merge` as default

**Comment by prologic** [comment]
> btw @prologic i suggest to make `squash and merge` as default

On teh repo? Can do :)

**Pull request merged**